### PR TITLE
Add multi get functions

### PR DESCRIPTION
**Pull request by nickdaugherty**
Adds functions for retrieving multiple documents at once, to compliment the existing functions for single document retrieval.

This can be used to grab a batch of documents to improve performance when retrieving large numbers of documents is necessary.

### CODE REVIEW

**Comment by nickdaugherty** [comment]
Tests failing upstream...this PR is missing tests of it's own (EP doesn't seem to have tests for the "single" versions of these functions).

File: includes/classes/Elasticsearch.php [hunk 1]
```diff
@@ -597,6 +597,51 @@ public function get_document( $index, $type, $document_id ) {
...
+					} else {
+						$docs[] = null;
+					}
+				}
+			}
+
+			return $docs;
+		}
+
+		return false;
```
Comment (brandon-m-skinner) [inline includes/classes/Elasticsearch.php]:
```suggestion
		if ( is_wp_error( $request ) ) {
			return false;
		}
		
		$response_body = wp_remote_retrieve_body( $request );

		$response = json_decode( $response_body, true );

		$docs = [];

		if ( is_array( $response['docs'] ) ) {
			foreach ( $response['docs'] as $doc ) {
				if ( ! empty( $doc['exists'] ) || ! empty( $doc['found'] ) ) {
					$docs[] = $doc['_source'];
				} else {
					$docs[] = null;
				}
			}
		}

		return $docs;
```

Comment (brandon-m-skinner) [reply includes/classes/Elasticsearch.php]: @chrean What are your thoughts?

Comment (nickdaugherty) [reply includes/classes/Elasticsearch.php]: I do like this approach more (return early for error), I had just copy/pasted the existing `get_document()` function :D. Let's go with your proposed change.

**Comment by brandon-m-skinner** [comment]
r826-stacks

**Pull request merged**